### DISCUSSION TITLE
🌳XSOSI 计划 10 月份数据

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#210] [2️⃣ 聊项目]
接 XSOSI 计划

- [6 月份数据](https://github.com/orgs/X-lab2017/discussions/115)
- [7 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/149)
- [8 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/173)
- [9 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/196)

10 月份数据即将出炉 @frank-zsy ~

几点猜想：总体活跃度应该会继续增长，Hypercrx 项目会相对更活跃与更具影响力（中国开源年会）~

以及可以根据这几个月的体验，构建组织层面的可视化看板了，看全体、以及不同两个组织下的项目、人员等要素的发展趋势~

**Comment by will-ww** [comment]
本月数据新鲜出炉，在 Hypercrx 项目的带动下，整体增长率为 12.20%，奖金池达到了 13800 ~

详细数据如下，大家可以看看，有没有遗漏的地方，可以随时提出~

<img width="798" alt="d63d7c319c186c6f71692a9192f85ec" src="https://user-images.githubusercontent.com/15010826/200224313-3a5de6b8-40e2-4064-9506-d6b70886e94a.png">

<img width="1394" alt="49aaaae5ef391fa15dffd9641d6ac41" src="https://user-images.githubusercontent.com/15010826/200224339-785e5f9d-1d31-4f2e-b619-04785e44d402.png">

**Reply by bifenglin** [reply]
tyn赞！

**Comment by will-ww** [comment]
从本月开始，校内同学的费用直接走同学们的银行卡了，主要考虑税费上的一些优惠，校外人员，还是直接微信转账的形式。
